### ISSUE TITLE
[testdisk] Feature: Progress bar

### ISSUE THREAD

**Issue by nwrox** [cgsecurity/testdisk#38]
When you select to copy one or more files there is no indication of current progress or estimated time. Would be nice to have a progress bar like in the wget.

![wget1](https://user-images.githubusercontent.com/26011383/33229402-3885caf2-d1b5-11e7-97d6-78d403b63f1c.png)

**Comment by cgsecurity** [comment]
There is an indication of progress: the number of files copied is updated in realtime.

**Comment by nwrox** [comment]
Sorry there is a indication as you said but I think that the number of copied files is not enough. If a progress bar like in the wget is too much is possible at least a eta count?

![testdisk](https://user-images.githubusercontent.com/26011383/33290160-ffc5f2dc-d3a8-11e7-8f0d-9c208c5d73e6.png)

**Comment by timofonic** [comment]
@cgsecurity What's wrong with a simple progress bar?

@nwrox I agree!

**Comment by cgsecurity** [comment]
The number of files to copy is unknown when the copy starts.
Counting the number of files before starting the copy may waste some time but the real problem is that the filesystem is not reliable, the disk may be damaged and parsing the directory structure may never complete.
IMO it's better to recover some files than recovering nothing because the program tries to count the number of files to copy...

**Comment by nwrox** [comment]
@cgsecurity you say that the number of files to copy is unknown, but you must select the files that you want to copy right?

**Comment by cgsecurity** [comment]
If you are undeleting files from a FAT32 and select a single directory to recover, there may be a lot of files inside...

**Comment by nwrox** [comment]
Got your point, this can become expensive programmatically, so i think there's no way to add this feature, this issue can be closed now, thanks.
